scripts/sync-gsuite-users.js

```diff
@@ -0,0 +1,150 @@
+
+// PART 1: get all gsuite under /individuals, and for each one make a query to core
+
+const gsuite_obj = [
+  // {
+  //   "primaryEmail": name.surname@aegee.eu, // from gsuite
+  //   "recoveryEmail": name.surname@gmail.com, // from gsuite
+  //   "g_surname": item.name.familyName, // surname coming from gsuite
+  //   "m_possible_users": [], // heuristic of people with the same surname in myaegee
+  //   "allocated": false, // whether we have found this person on myaegee
+  // } ;
+];
+
+//GAM tells me there are 800-900 users, so we need two invocations.
+
+const superagent = require('superagent');
+
+(async () => {
+
+  try{
+
+    const part1 = await superagent.get('gsuite-wrapper:8084/accounts?max=500')
```

Comment:
await request({
        url: 'gsuite-wrapper:8084/accounts?max=500',
        method: 'GET'});

I think this would work and remove the need for installing superagent

Reply:
const request = require('request-promise-native');

At the top

Reply:
done and tested (not tested with a prod db though, just tested the translation and that the requests don't error)